tests/test_meteodatalab/test_relhum.py

```diff
@@ -1,20 +1,56 @@
 # Third-party
+import pytest
 from numpy.testing import assert_allclose
 
 # First-party
 from meteodatalab.grib_decoder import GribReader
 from meteodatalab.operators.relhum import relhum
 
+expected_water = {
+    "centre": "lssw",
+    "paramId": 500037,
+    "shortName": "RELHUM",
+    "units": "%",
+    "name": "Relative Humidity",
+}
 
-def test_relhum(data_dir, fieldextra):
+expected_ice = {
+    "centre": "lssw",
+    "paramId": 503195,
+    "shortName": "RH_ICE",
+    "units": "%",
+    "name": "relative humidity over ice",
+}
+expected_mixed = {
+    "centre": "lssw",
+    "paramId": 503078,
+    "shortName": "RH_MIX_EC",
```

Comment:
I think this is not correct. It sets the parameter number to 220 which has no relation to the relative humidity.

Reply:
As discussed, let's go ahead with this. The code is referenced in fieldextra cosmo dictionary

Reply:
Correct, `RH_MIX_EC` is defined in a local code here: https://github.com/COSMO-ORG/fieldextra/blob/develop/resources/dictionary_cosmo.txt#L2038 and points to parameter 220 which corresponds to  `Minimum specific humidity at 2m` here: https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_table4-2-0-1.shtml